docs/getting_started.md

```diff
@@ -39,25 +36,58 @@ traces.
 -   Use open source libraries with OpenTracing integration to get
     vendor-neutral instrumentation for free
 
-### Prerequisites
+###Prerequisites
 
 -   You need Go 1.7 or higher installed on your machine.
 -   Requires a running Jaeger backend to view the traces.
-    -   See [Running Jaeger backend in docker](#running-jaeger-in-docker-container)
+    -   See [Running Jaeger backend in docker](#docker)
 
-### Installation
+###Running 
 
-```shell 
+```bash
 go get github.com/uber/jaeger
+cd $GOPATH/src/github.com/uber/jaeger
 make install_examples
+cd examples/hotrod
+go run ./main.go all
 ```
 
-### Running
+Then navigate to `http://localhost:8080`
 
-```shell
-cd examples/hotrod
-go run ./main.go all
+##Client Libraries
```

Comment:
why mention clients here at all? If you look at zipkin.io, they have a top level document about instrumentation.